src/v/model/tests/random_batch.h

```diff
@@ -36,6 +36,9 @@ struct record_batch_spec {
     std::optional<model::timestamp> timestamp;
 };
 
+// Makes an random alphanumeric string, encoded in an iobuf.
+iobuf make_iobuf(size_t);
```

Comment:
can we change the name of this. it looks like it had been intended to be internal to the translation unit, but now at the global scope the name isn't helpful.

Reply:
Isn't the effective name here `random_generators:make_iobuf`?

Reply:
it looks to me like the fully qualified name is `model::test::make_iobuf`. i'd move it to random generators or bytes test helpers.

Reply:
Ah, I was confusing `using-directive` with `using-declaration` behavior here. Will do, thanks.